test/integration/ioredis/docker-compose.yml

```
@@ -0,0 +1,155 @@
+version: "3.8"
+
+services:
+  redis-master:
+    image: redis:latest
+    container_name: redis-master
+    hostname: redis-master
```

"main" and "workers" is the preferred terminology

```suggestion
  redis-main:
    image: redis:latest
    container_name: redis-main
    hostname: redis-main
```